packit_service/worker/handlers/forges.py

```diff
@@ -83,7 +83,8 @@ def run(self) -> TaskResults:
                     " field in the settings of the FAS account (if you don't have it set already)"
                     f" and provide it in a comment in this issue as "
                     f"`{self.service_config.comment_command_prefix} verify-fas "
-                    "<my-fas-username>`. We automatically check for the match between the `GitHub"
+                    "<my-fas-username>` (please, also make sure your profile is not private). "
```

Comment:
We already have one "please" in the sentence and the whole message is quite long so every saved word counts.

```suggestion
                    "<my-fas-username>` (make sure your profile is not private). "
```